github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1293,3 +1293,65 @@ periodics:
       - name: pgp-bot-key
         secret:
           secretName: pgp-bot-key
+- name: periodic-kubevirt-e2e-k8s-1.21-sig-performance
```

Comment:
If you want you could also add an optional presubmit PR, to e.g. test perf changes already now on PRs, without thinking about capacity.

Reply:
Make sense, I will create it later!

Reply:
If we add this test for each PR, we probably do not need to run daily.

But, let's start with this test and update it later.
The goal is to run daily a medium size test, e.g., creating 500 to 1000VMs.